### PR TITLE
Removing the Unused Imports

### PR DESCRIPTION
**Pull request by SandeepSimha**
## Description

Removed the Unused imports and Refactored the code.

## Related Issues
None 

## Tests

I added the following tests:

It does not affect any of the test cases.

## Checklist

Before you create this PR confirm that it meets all requirements listed below by checking the relevant checkboxes (`[x]`). This will ensure a smooth and quick review process.

- [x] I read the [Contributor Guide] and followed the process outlined there for submitting PRs.
- [x] I signed the [CLA].
- [x] I read and followed the [Flutter Style Guide], including [Features we expect every widget to implement].
- [x] I read the [Tree Hygiene] wiki page, which explains my responsibilities.
- [x] I updated/added relevant documentation (doc comments with `///`).
- [x] All existing and new tests are passing.
- [x] The analyzer (`flutter analyze --flutter-repo`) does not report any problems on my PR.
- [x] I am willing to follow-up on review comments in a timely manner.

## Breaking Change

Did any tests fail when you ran them? Please read [Handling breaking changes].

- [x] No, no existing tests failed, so this is *not* a breaking change.
- [ ] Yes, this is a breaking change. *If not, delete the remainder of this section.*
   - [ ] I wrote a design doc: https://flutter.dev/go/template *Replace this with a link to your design doc's short link*
   - [ ] I got input from the developer relations team, specifically from: *Replace with the names of who gave advice*
   - [ ] I wrote a migration guide: *Replace with a link to your migration guide*

[issue database]: https://github.com/flutter/flutter/issues
[Contributor Guide]: https://github.com/flutter/flutter/wiki/Tree-hygiene#overview
[Test Coverage]: https://github.com/flutter/flutter/wiki/Test-coverage-for-package%3Aflutter
[Flutter Style Guide]: https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo
[Features we expect every widget to implement]: https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo#features-we-expect-every-widget-to-implement
[CLA]: https://cla.developers.google.com/
[Tree Hygiene]: https://github.com/flutter/flutter/wiki/Tree-hygiene
[Handling breaking changes]: https://github.com/flutter/flutter/wiki/Tree-hygiene#handling-breaking-changes

### CODE REVIEW

**Comment by SandeepSimha** [comment]
I signed it!

**Comment by jmagman** [comment]
\cc @blasten

File: dev/integration_tests/android_host_app/app/src/main/java/io/flutter/add2app/MainActivity.java [hunk 1]
```diff
@@ -5,7 +5,9 @@
 package io.flutter.add2app;
 
 import android.os.Bundle;
+
```
Comment (blasten) [inline dev/integration_tests/android_host_app/app/src/main/java/io/flutter/add2app/MainActivity.java]: Thanks for the PR. Did you use a tool to auto-format the Java files?

Comment (SandeepSimha) [reply dev/integration_tests/android_host_app/app/src/main/java/io/flutter/add2app/MainActivity.java]: Yes, Android studio. I did it using "Optimize Imports".

**Comment by blasten** [comment]
It looks like commit https://github.com/flutter/flutter/pull/48863/commits/917101d8ad73a9d4733080847565a0410a0ff113 was authored by a user with a different email address than the one @SandeepSimha used to open this PR.  Would you mind visiting https://cla.developers.google.com/ and signing the CLA using this email address?

**Comment by matthew-carroll** [comment]
@SandeepSimha is there any particular reason that organizing imports was important? I think it would be preferable to just remove the lines that need to be removed, otherwise we end up with a commit that looks much larger than it is, and it pollutes the readability of the commit history. Would you mind trying again with just the removal of unused import lines?

**Comment by blasten** [comment]
@matthew-carroll I don't have strong opinions, but if you do, please make the final call.

**Comment by blasten** [comment]
@SandeepSimha please sign the CLA on https://cla.developers.google.com/

**Comment by matthew-carroll** [comment]
@blasten if you mean an opinion about minimal changes, I do think this PR should be reduced to only removing unused import lines. I don't think we should reorganize imports if we can help it.

**Comment by jonahwilliams** [comment]
Because of the lack of activity on this PR I'm going to close it to remove from our review queue. Please feel free to send us a new PR or discuss in an issue anytime.

**Pull request closed without merging**